gems/sorbet-runtime/lib/types/props/private/serde_transform.rb

```diff
@@ -65,6 +65,14 @@ def self.generate(type, mode, varname)
           raw = type.raw_type
           if NO_TRANSFORM_TYPES.any? {|cls| raw <= cls}
             nil
+          elsif raw <= Float
+            case mode
+            when Deserialize then "#{varname}.to_f"
+            when Serialize then nil
+            else T.absurd(mode)
+            end
+          elsif raw <= Numeric
+            nil
```

Comment:
I wasn't super clear on the distinction between `serialize` and `deserialize` here. In my mind, I was pretty sure that I only wanted to call `.to_f` when deserializing, but nearly all the code in this file was agnostic of mode, which I found kind of surprising.

I'm happy to call `.to_f` in both places, but it would mean I'd need to add another method to a whitelist.